### PR TITLE
[MCH] propagate tracks to MID

### PR DESCRIPTION
**Pull request by pillot**
This is the first step toward matching MCH and MID tracks. It propagates the MCH track parameters and covariances at the last attached cluster to the first MID chamber, where the matching will be performed, and store them in the TrackMCH output object.

### CODE REVIEW

File: Detectors/MUON/MCH/Tracking/src/TrackExtrap.cxx
Comment (aphecetche) [inline Detectors/MUON/MCH/Tracking/src/TrackExtrap.cxx:252]: why do you pass a pointer (which you anyway don't test for nullptr) instead of a (const) reference ?

Comment (pillot) [reply Detectors/MUON/MCH/Tracking/src/TrackExtrap.cxx:252]: just for consistency with all other extrapolation methods in this class (inherited from aliroot). A reference (not const since the parameters are modified) might be preferable indeed, but if I change that, I would prefer to change consistently all the methods (in a separate commit).

Comment (aphecetche) [reply Detectors/MUON/MCH/Tracking/src/TrackExtrap.cxx:252]: OK. Would be nice to have references instead indeed. When you say separate commit you mean an additional commit in this PR or another PR altogether ?

Comment (pillot) [reply Detectors/MUON/MCH/Tracking/src/TrackExtrap.cxx:252]: I was thinking another PR, as it requires changes in other places unrelated to this PR.

Comment (aphecetche) [reply Detectors/MUON/MCH/Tracking/src/TrackExtrap.cxx:252]: ok, then I will merge this one as soon as I've access to the green merge button...

**Review by aphecetche — APPROVED** [review]

**Pull request merged**
